### DISCUSSION TITLE
Cronicle path base

### DISCUSSION THREAD

**Discussion by dinobolognese** [jhuckaby/Cronicle#432] [Ideas]
HI,

is there any way to setup a path base for cronicle?
i mean something like that: http://localhost/cron, so every resources are expose with this path, 
/cron/api/*
/cron/js/* 
/cron/css/*
...

It would be usefull to have a parameter in WebServer config properties.

Thanks.
Dino

**Comment by mikeTWC1984** [comment]
I think there is no way to achieve that via config. But it's fairly straight forward via reverse proxy, e.g. nginx. You can browse recent issues, it's quite popular topic.
